src/pass/AnnotationHandler.cpp

```diff
@@ -266,6 +266,20 @@ static bool isLambdaFunction(const Function &F) {
 
 AnnotationHandler::AnnotationHandler(Module &M) : M(M), Types(M) {}
 
+void AnnotationHandler::populateAnnotations(
+    DenseMap<Value *, GlobalVariable *> &StubToKernelMap) {
+
+  for (Function &F : M) {
+    if (F.isDeclaration())
+      continue;
+    if (!(isDeviceKernel(&F) || StubToKernelMap.contains(&F)))
+      continue;
+    if (hasAnnotateTag(F, "jit"))
```

Comment:
Do you want to keep pre-existing "jit" annotations that may have pre-existing RC parameters?